docs/src/modeler/time_series.md

```diff
@@ -69,33 +64,25 @@ Example:
     forecast = Deterministic("max_active_power", data, resolution, scaling_factor_multiplier = get_max_active_power)
 ```
 
-In this example the forecasted component is a generator. Whenever the user
-retrieves the forecast data PowerSystems will call
-`get_max_active_power(component)` and multiply the result with the forecast
-values (scaling factors).
+In this example, the forecasted component is a generator. Whenever the user retrieves the forecast data PowerSystems will call `get_max_active_power(component)` and multiply the result with the forecast values (scaling factors). For instance it the maximum active power returns the value 50.0 and the scaling factor at some time point is 0.65, the forecast value will correspond to 32.5. 
 
 ## Adding time series to the System
-Adding time series data to a system requires a component that is already
-attached. Extending the example above:
+Adding time-series data to a system requires a component that is already attached to the system. Extending the example above:
```

Comment:
Markdown ignores newlines like this in the rendered file. I automatically formatted the text with Vim.  IMHO, adding line breaks makes the raw file easier to read.

Reply:
We should run the formatter at the end. I am editing in the GitHub editor to make it easier.